graphdatascience/pipeline/nr_training_pipeline.py

```diff
@@ -9,24 +9,61 @@
 
 class NRTrainingPipeline(TrainingPipeline[NRModel]):
     def addLinearRegression(self, **config: Any) -> "Series[Any]":
+        """
+        Add a linear regression model to the pipeline.
+
+        Args:
+            **config: The configuration for the linear regression model.
+
+        Returns:
+            The result of the query.
+
+        """
         query = f"{self._query_prefix()}addLinearRegression($pipeline_name, $config)"
         params = {"pipeline_name": self.name(), "config": self._expand_ranges(config)}
 
         return self._query_runner.run_query(query, params).squeeze()  # type: ignore
 
     def addRandomForest(self, **config: Any) -> "Series[Any]":
+        """
+        Add a random forest regressor to the pipeline.
```

Comment:
probably we want to align the language here that this is also a candidate (applies to the other pipelines as well)